pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -1686,7 +1689,7 @@ public PulsarClientImpl createClientImpl(ClientConfigurationData clientConf)
                 .timer(brokerClientSharedTimer)
                 .internalExecutorProvider(brokerClientSharedInternalExecutorProvider)
                 .externalExecutorProvider(brokerClientSharedExternalExecutorProvider)
-                .scheduledExecutorProvider(brokerClientSharedScheduledExecutorProvider)
```

Comment:
why is `scheduledExecutorProvider` removed?

Reply:
Good catch, reverted.